### ISSUE TITLE
[Error: null pointer: getValidShotByDoseNumber(2).getAdministrationDate()]

### ISSUE THREAD

**Issue by vshahrda** [cdsframework/ice#38]
Hello, I am getting this error for one of my requests.

Full response:
```
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
    <soap:Body>
        <soap:Fault>
            <faultcode>soap:Server</faultcode>
            <faultstring>org.omg.dss.DSSRuntimeExceptionFault: OpenCDS call to Drools.execute failed with error: [PrimaryProcess:1 - Initialize Reference Data:3] -- [PrimaryProcess:1 - Initialize Series and Seasons:11] -- [PrimaryProcess:1 - Identify Candidate Series:5] -- [PrimaryProcess:1 - Identify Candidate Doses:10] -- [PrimaryProcess:1 - Record Disease Immunity:9] -- [PrimaryProcess:1 -  Evaluate History Against All Series:6] -- [PrimaryProcess:1 - Select Best Series:8] -- [Error: null pointer: getValidShotByDoseNumber(2).getAdministrationDate()]
[Near : {... getValidShotByDoseNumber(2).ge ....}]
             ^
In Rule "SeriesSelection.SelectHibOMPSeriesIfTwoDosesHibOMPOneBefore7MonthsOtherBefore12Months" in knowledgeModule/gov.nyc.cir.ice/gov.nyc.cir^ICE^1.0.0^SeriesSelection.drl</faultstring>
        </soap:Fault>
    </soap:Body>
</soap:Envelope>
```

To reproduce: Execute the attached request, which would produce this error. Any idea what might be causing this?

[failing-request.zip](https://github.com/cdsframework/ice/files/8779038/failing-request.zip)

**Comment by vshahrda** [comment]
[failing-request.zip](https://github.com/cdsframework/ice/files/8779044/failing-request.zip)

**Comment by vshahrda** [comment]
Does anyone know how I can troubleshoot this? I'm still confused by this error.

**Comment by darylbc** [comment]
A fix is in place for the next release, 1.35.1. We expect to make this publicly available tomorrow.
